packages/lesswrong/server/callbacks/alignment-forum/callbacks.ts

```diff
@@ -15,7 +15,7 @@ export const recalculateAFBaseScore = async (document: VoteableType): Promise<nu
     afPower: {$exists: true},
     cancelled: false,
   }).fetch()
-  return votes ? votes.reduce((sum, vote) => { return vote.afPower ?? 0 + sum}, 0) : 0
+  return votes.reduce((sum, vote) => { return (vote.afPower ?? 0) + sum}, 0) : 0
```

Comment:
Forgot to remove the second half of the ternary

Reply:
Wasn't actually intending to remove the ternary (though I guess it's unnecessary), I was taking out a console-log I had stuck in between the calculation and the return and got an extra token. Just pushed a fixed commit.